modules/jdbc/src/main/java/org/apache/ignite/jdbc/JdbcPreparedStatement.java

```diff
@@ -107,7 +106,7 @@ public int[] executeBatch() throws SQLException {
 
         closeResults();
 
-        if (CollectionUtils.nullOrEmpty(batchedArgs) || StringUtil.isNullOrEmpty(sql)) {
+        if (CollectionUtils.nullOrEmpty(batchedArgs) || (sql == null || sql.isBlank())) {
```

Comment:
Perhaps, it would be better to reuse `org.apache.ignite.internal.util.StringUtils#nullOrBlank` from core module

Reply:
done